gradle.properties

```diff
@@ -1,3 +1,3 @@
-release_version = 4.0.0
-
-docker_image_name=
+release_version=5.1.0
+app_main_class=com.exactpro.th2.codec.MainKt
```

Comment:
Is this used by our custom plugin? Could you please take a look at this and leave a comment here in the file?

Reply:
Looks like 'docker_image_name' is not used. I've removed it.

Reply:
But what about `app_main_class`? I was asking about that property

Reply:
It used here: https://github.com/th2-net/th2-gradle-plugin/blob/dev/src/main/groovy/com.exactpro.th2.docker-conventions.gradle

Reply:
Okay, got it. Thank you for checking. I would like to add a comment about that. Because it is not common to have a property and don't use it in our *.gradle files